include/aspect/initial_temperature/patch_on_S40RTS.h

```diff
@@ -0,0 +1,115 @@
+/*
+  Copyright (C) 2011 - 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_initial_temperature_patch_on_S40RTS_h
+#define _aspect_initial_temperature_patch_on_S40RTS_h
+
+#include <aspect/initial_temperature/interface.h>
+#include <aspect/initial_temperature/S40RTS_perturbation.h>
+#include <aspect/simulator_access.h>
+#include <aspect/utilities.h>
+
+
+namespace aspect
+{
+  namespace InitialTemperature
+  {
+    using namespace dealii;
+
+    /**
+     * A class that implements a prescribed temperature field determined from
+     * a Vs model given by a PatchOnS40RTS input file.
```

Comment:
Please add that below a certain depth, the S40RTS model is used. And is `PatchOnS40RTS input file` a search and replace typo?